x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -282,8 +282,8 @@ export const AlertsTableComponent: FC<DetectionEngineAlertTableProps> = ({
         id: tableId,
         title: i18n.SESSIONS_TITLE,
         defaultColumns: finalColumns.map((c) => ({
-          ...c,
           initialWidth: DEFAULT_COLUMN_MIN_WIDTH,
+          ...c,
```

Comment:
Was this change intended?

Reply:
@michaelolo24 

yes.. previously, if `c` has an initialWidth (we have hardcoded some values of initialWidth), it was being overwritten by `DEFAULT_COLUMN_MIN_WIDTH`.  Now, we overwrite default value if there some value in `c i.e finalColumns`

Reply:
ahh, okay. Sorry missed that one, thanks!